ftd/src/p1/parser.rs

```diff
@@ -872,9 +872,70 @@ fn clean_line(line: &str) -> String {
     if line.starts_with("\\;;") || line.starts_with("\\-- ") {
         return line[1..].to_string();
     }
+
+    if !line.contains(" ;; <hl>") {
```

Comment:
Just change it to this

```suggestion
    // Ignoring <hl> to ignore code highlighting comments
    // todo: need to fix the code highlighting syntax
    if !line.contains("<hl>") {
```